connectors/sources/sharepoint_server.py

```diff
@@ -480,16 +480,16 @@ def get_default_configuration(cls):
             },
             "ssl_enabled": {
                 "display": "toggle",
-                "label": "Enable SSL",
+                "label": "Enable SSL certificate verification",
                 "order": 5,
                 "type": "bool",
-                "value": False,
+                "value": True,
```

Comment:
Why True by default? It's always on normally?

Reply:
yep, it's a bad practice to _not_ verify certificates – you'd only want to do that under special circumstances, otherwise you're circumventing one of the core benefits of what TLS provides